omero_search_engine/api/v1/resources/utils.py

```diff
@@ -1543,6 +1555,22 @@ def delete_container(ids, resource, data_source, update_cache, synchronous_run=F
     es = search_omero_app.config.get("es_connector")
     ids = ids.split(",")
     for id in ids:
+        no_images = get_number_of_images_inside_container(resource, data_source, id)
+        if no_images == 0:
+            search_omero_app.logger.info(
+                "The requested %s with ID=%s does not correspond to any existing one."
+                % (resource, id)
+            )
+            return
+        elif no_images > 740000 and synchronous_run:
+            # This number has been approximately estimated through trial and error.
+            search_omero_app.logger.info(
+                "Due to the high number of images in the %s with ID=%s, "
+                "an asynchronous delete operation is highly advised. "
+                "\nInitiating asynchronous deletion." % (resource, id)
+            )
+            synchronous_run = False
```

Comment:
Following today's discussion, We may want to exit here and ask the user to review the config
i.e. option 1